Zend/tests/operator_overloads/equals_fallback_comparison.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+operator overload: equals fallback to comparison operator
+--FILE--
+<?php
+
+class A {
+    public int $value;
+
+    public operator <=>(mixed $other): int
+    {
+		return ($this->value <=> $other);
```

Comment:
Nit: indent

Reply:
Was being confused because the file view was showing them misalligned, but it's because you're mixing tabs and spaces, PHPT files should be space indented.